pandas/tools/plotting.py

```diff
@@ -2262,11 +2262,12 @@ def result(self):
 
 def _plot(data, x=None, y=None, subplots=False,
           ax=None, kind='line', **kwds):
+
     kind = _get_standard_kind(kind.lower().strip())
     if kind in _all_kinds:
         klass = _plot_klass[kind]
     else:
-        raise ValueError('Invalid chart type given %s' % kind)
+        raise ValueError("%r is not a valid plot kind" % kind)
```

Comment:
a very small thing, but can you add `'` around the `%r`? I think that makes the message more clear